microsoft-edge/devtools-guide-chromium/device-mode/testing-other-browsers.md

```diff
@@ -50,6 +51,7 @@ Firefox has a [responsive design view](https://developer.mozilla.org/docs/Tools/
 
 <!-- ------------------------------ -->
 #### EdgeHTML emulation
```

Comment:
Delete "EdgeHTML" section & point to MDN instead? 
@zoherghadyali @bwalderman 

Rendered article section: 
pending

Reply:
Yes, let's delete this whole section. No need to point to MDN. No scenario mentioned in this section is valid anymore.